crates/radix-engine-toolkit/src/functions/transaction_v1/notarized_transaction.rs

```diff
@@ -64,3 +64,15 @@ pub fn statically_validate(
         .and_then(|prepared| validator.validate_notarized_v1(prepared))
         .map(|_| ())
 }
+
+pub fn extract_signer_public_keys(
```

Comment:
We should probably adjust the name or add a comment that this returns an error if the transaction is invalid according to the most recent validation logic.

... and ditto on the method exposed from Uniffi.

And we should flag this to Pawel that he should `try/catch` it :)

Reply:
Sounds good, I will mention this in a comment.

Reply:
Also, hmmm... maybe we should fix the validation rules, perhaps to the babylon rules for v1 and cuttlefish rules for v2?